### PR TITLE
Add support for ppc64le

### PR DESCRIPTION
**Pull request by amitsadaphule**
Crossplane build and tests have been validated on ppc64le with these changes.

### CODE REVIEW

**Comment by negz** [comment]
@AaronME would you mind taking a look at this PR, since you recently did a bunch of work with architecture support? I'm happy to merge if it looks good to you.

**Review by AaronME — CHANGES_REQUESTED** [review]
LGTM, minor notes.

**Comment by negz** [comment]
Thanks @AaronME! @amitsadaphule let me know when this is ready to merge.

File: makelib/common.mk [hunk 1]
```diff
@@ -126,10 +126,14 @@ SAFEHOSTARCH := amd64
 TARGETARCH := amd64
 endif
 
-ifeq ($(filter amd64 arm64 ,$(SAFEHOSTARCH)),)
+ifeq ($(filter amd64 arm64 ppc64le ,$(SAFEHOSTARCH)),)
 $(error build only supported on amd64 and arm64 host currently)
```
[the remark this answers is not in the document]
Comment (amitsadaphule) [reply makelib/common.mk:130]: Right, missed that. I'll add that.

File: makelib/common.mk [hunk 2]
```diff
@@ -126,10 +126,14 @@ SAFEHOSTARCH := amd64
...
 endif
 
+ifeq ($(HOSTARCH),ppc64le)
+		PLATFORMS = linux_ppc64le
+endif
+
```
[the remark this answers is not in the document]
Comment (amitsadaphule) [reply makelib/common.mk:136]: OK, thanks for the pointer! :) I'll remove this and add in the main project being compiled.

File: makelib/common.mk (hunk 1 above)
[the remark this answers is not in the document]
Comment (amitsadaphule) [reply makelib/common.mk]: Done.

File: makelib/common.mk (hunk 2 above)
[the remark this answers is not in the document]
Comment (amitsadaphule) [reply makelib/common.mk]: Done.

**Comment by amitsadaphule** [comment]
@negz I've incorporated the review comments and validated the changes locally. Ready to merge! :) Thanks!

**Comment by amitsadaphule** [comment]
@negz @AaronME please review.

**Review by AaronME — APPROVED** [review]
LGTM!

Crossplane builds successfullly on mac m1 and intel processors.

**Comment by negz** [comment]
Thanks for adding this functionality @amitsadaphule!

**Pull request merged**